hardhat.config.js/solidity.js

```diff
@@ -1,16 +1,13 @@
-const compilerSettings = process.env.ENABLE_OPTIMIZER
-  ? { optimizer: { enabled: true, runs: 200 } }
-  : {};
+const compilerSettings = process.env.ENABLE_OPTIMIZER ? { optimizer: { enabled: true, runs: 200 } } : {};
 
 const compilers = {
+  '0.5.7': { settings: compilerSettings, version: '0.5.7' }, // nexus mutual v1
   '0.5.17': { settings: compilerSettings, version: '0.5.17' }, // nexus mutual v1
   '0.8.9': { settings: compilerSettings, version: '0.8.9' }, // nexus mutual v2
 };
 
 const contracts = {
-  '0.5.7': [
-    'contracts/modules/governance/Governance.sol',
-  ],
+  '0.5.7': ['contracts/modules/governance/Governance.sol'],
```

Comment:
Please revert the modifications to the lines that weren't modified.

Reply:
Formatted according to the rules found in `.prettierrc`.